### PR TITLE
DM-36092: ctrl_bps_parsl should have doc and tests directories

### PR DESCRIPTION
**Pull request by PaulPrice**

### CODE REVIEW

**Comment by timj** [comment]
@PaulPrice  What's your status on this?

**Comment by PaulPrice** [comment]
I have two important projects sitting ahead of it in my queue, but I've already made a lot of progress. I don't remember what's outstanding; maybe the docs conversion?

**Comment by PaulPrice** [comment]
Oh, I was hoping to include some tests. That would involve some real work to mock things up.

**Comment by timj** [comment]
Do you want to put the tests on a separate ticket? Then we can benefit from the other cleanups whilst the tests are being developed?

**Comment by timj** [comment]
Add simplest possible test here. Get a coverage report. Then call it done and do proper test in new ticket. The cleanups here seem like they should be merged because I'm worried that we are about to do some new feature work to the plugin and it would be helpful for this to be in there first.

**Comment by PaulPrice** [comment]
@timj Does this look ready for review?

File: .github/workflows/build_docs.yaml
Comment (timj) [inline .github/workflows/build_docs.yaml:33]: You don't need these or the ones below to build the docs -- those are for butler optional classes.

File: .github/workflows/build.yaml
Comment (timj) [inline .github/workflows/build.yaml:55]: If you aren't using `-n` option here you can drop the xdist install above.

File: LICENSE
Comment (timj) [inline LICENSE:1]: You can probably just say:
```
See the COPYRIGHT file for copyright information.
```
like we do elsewhere. I'm not entirely sure this license can be right given that BPS is GPL.

File: doc/changes/README.rst
Comment (timj) [inline doc/changes/README.rst:16]: I think I decided that the distinction between other and misc was meaningless so I'm trying to standardize on misc.

**Review by timj — APPROVED** [review]
This looks like a good start to me.

**Comment by timj** [comment]
Also, once this is merged you can add ctrl_bps_parsl as a dependency of pipeline_lsst_io so that you get the docs integrated into the web site properly.

File: LICENSE
Comment (PaulPrice) [reply LICENSE:1]: The license is as inherited from the DESC gen3_workflow package.

Comment (timj) [reply LICENSE]: It's not compatible with BPS though so it can't be BSD license.

I'd like to convert all the middleware to BSD but that needs paperwork.

**Comment by timj** [comment]
Didn't we decide to merge this PR so that it wouldn't complicate people doing future work?

**Comment by PaulPrice** [comment]
My understanding is that it's blocked on RFC-889 (that's what the ticket says).

**Comment by timj** [comment]
I think RFC-889 decided that (1) ctrl_bps_parsl has to be at least GPL because it depends directly on GPL code (2) we will make it dual license BSD + GPL.  So I think if you make it GPL you can then merge. I will be adopting RFC-889 very soon and then have to add BSD bottom up and will change this package at that point.

**Pull request merged**
